Cargo.toml

```diff
@@ -1,11 +1,11 @@
 [package]
 name = "tree-sitter-scala"
 description = "scala grammar for the tree-sitter parsing library"
-version = "0.20.0"
+version = "0.22.0"
```

Comment:
Should this be `0.21.0`?

Reply:
Addressed in Addressed in 4c7275c70e41c7c1fad6f4e52a60dd3941db5bba